extensions/interactions/GraphInput/validator.js

```diff
@@ -53,7 +57,41 @@ oppia.filter('oppiaInteractiveGraphInputValidator', [
     warningsList = warningsList.concat(
       baseInteractionValidationService.getAllOutcomeWarnings(
         answerGroups, defaultOutcome, stateName));
+    for (var i = 0; i < answerGroups.length; i++) {
+      var ruleSpecs = answerGroups[i].rule_specs;
+      for (var j = 0; j < ruleSpecs.length; j++) {
+        var ruleSpec = ruleSpecs[j];
+        try {
+          if (ruleSpec.rule_type === 'IsIsomorphicTo' &&
+              ruleSpec.inputs.g.vertices.length > ISOMORPHISM_VERTICES_LIMIT) {
+            warningsList.push({
+              type: WARNING_TYPES.CRITICAL,
+              message: 'The graph used in the rule ' + (j + 1) +
+                       ' in group ' + (i + 1) +
+                       ' exceeds supported maximum number of vertices of ' +
+                        ISOMORPHISM_VERTICES_LIMIT + ' for isomorphism check.'
+            });
+          }
 
+          if (ruleSpec.inputs.g.vertices.length > VERTICES_LIMIT) {
+            warningsList.push({
+              type: WARNING_TYPES.CRITICAL,
+              message: 'The graph used in the rule ' + (j + 1) +
+                       ' in group ' + (i + 1) +
+                       ' exceeds supported maximum number of vertices of ' +
+                        VERTICES_LIMIT + '.'
+            });
+          }
+        } catch (e) {
```

Comment:
Hi @seanlip Is it standard?
 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch
Here, there is no supported syntax for a **catch** without parameters?